docs/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md

```diff
@@ -120,9 +120,9 @@ A milestone reflects a state of a file in Web Modeler with a certain level of qu
 
 <!-- To be added once GA released:
```

Comment:
I assume this comment should be removed and the content added to the docs since this is now GA?

Reply:
Yes, thank you!